### ISSUE TITLE
vehicle.pos setter bug

### ISSUE THREAD

**Issue by Lekenthu** [altmp/altv-issues#2357]
### Description of the problem

Since the last alt:V update, it is not possible to teleport a vehicle when the player is in the vehicle, for example as a driver. The vehicle will appear in the new position for a moment and then teleport back to the old position the player.

### Reproduction steps

Example script:
```
if(target.vehicle) {
        target.vehicle.dimension = player.dimension;
        target.vehicle.pos = {x: player.pos.x+2, y: player.pos.y+2, z: player.pos.z};
        target.dimension = player.dimension;
}
```

### Expected behaviour

.

### Operating system

Windows 11

### Version

alt:V build #16.2.15, branch release

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest RC (Release Candidate) version available at the time of writing this ticket.*

**Comment by ashsrv** [comment]
Same issue, Windows 10

**Comment by BluePeer** [comment]
sounds similar but not identical

a streamed vehicle can't set the position (c# / js)
only outside stream range, but no visual move (if it is inside streamrange)
related to the test scenario it is the same issue idk currently

**Comment by akamorphe** [comment]
> sounds similar but not identical
> 
> a streamed vehicle can't set the position (c# / js) only outside stream range, but no visual move (if it is inside streamrange) related to the test scenario it is the same issue idk currently

**It was exemplified with video.**
https://youtu.be/SYj2i4pskzk

**Comment by Lekenthu** [comment]
Can this be expected to be rectified in the foreseeable future? A couple of my systems have become unusable since the update.

**Comment by deluvas1911** [comment]
Fixed with next rc/release (16.2.17)

**Comment by xx7eVeN** [comment]
I have this issue from time to time in 16.2.42 release (with debug on).
This sounds pretty similar to my approach as I am also changing dimensions for vehicle, player and teleporting the vehicle. :|
Im testing around for now if I can fix it or if something else causes this issue, maybe someone can check if he can reproduce this? :)
